101-azure-api-for-fhir/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the service."
+            }
+        },
+        "environmentName": {
+            "allowedValues": [
+                "global",
+                "gov"
+            ],
+            "type": "string",
+            "metadata": {
+                "description": "The type of environment to deploy the service"
+            }
+        },
+        "location": {
+            "defaultValue": "[resourceGroup().location]",
+            "type": "string",
+            "metadata": {
+                "description": "Location of Azure API for FHIR®"
+            }
+        }
+    },
+    "variables": {
+        "authenticationConfiguration": {
+            "global": {
+                "audience": ".azurehealthcareapis.com",
+                "authority": "https://login.microsoftonline.com/"
+            },
+            "gov": {
+                "audience": ".azurehealthcareapis.us",
+                "authority": "https://login.microsoftonline.us/"
+            }
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.HealthcareApis/services",
+            "apiVersion": "2019-09-16",
+            "name": "[parameters('serviceName')]",
+            "location": "[resourceGroup().location]",
```

Comment:
Instead of setting the location to resourcGroup().location, use the parameter named location. It defaults to the resource group location but also let's users override the value when needed.